app/services/submission_service.rb

```diff
@@ -22,13 +22,8 @@ def create_submission(
         if gravity_user_id
           User.find_or_create_by!(gravity_user_id: gravity_user_id)
         else
-          User.create!(
-            name: submission_params[:user_name],
-            email: submission_params[:user_email],
-            phone: submission_params[:user_phone]
-          )
+          User.create!
```

Comment:
I was thinking that we wouldn't even create a user for anonymous users and keep the `user_id` field on the submission as `nil`.